Description: Remove the special case for BSD in the bootstrap script.
 This is a simple brute-force patch until I figure out why that specific
 version of autoconf is needed on BSD.
Author: Tzafrir Cohen <tzafrir@debian.org>
Bug: https://issues.asterisk.org/view.php?id=17161
Bug-Debian: https://bugs.debian.org/624569
Forwarded-Upstream: no
Last-Update: 2016-03-19

--- a/bootstrap.sh
+++ b/bootstrap.sh
@@ -21,10 +21,6 @@
 		export AUTOCONF_VERSION=2.63
 		export AUTOMAKE_VERSION=1.9
 		;;
-	*'BSD'*)
-		MY_AC_VER=-2.62
-		MY_AM_VER=-1.9
-		;;
 	*'SunOS '*)
 		MY_AC_VER=
 		MY_AM_VER=-1.9
